onebusaway-android/src/main/java/org/onebusaway/android/report/ui/InfrastructureIssueActivity.java

```diff
@@ -0,0 +1,443 @@
+package org.onebusaway.android.report.ui;
+
+import android.content.Context;
+import android.content.Intent;
+import android.location.Address;
+import android.location.Geocoder;
+import android.location.Location;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.view.View;
+import android.view.Window;
+import android.widget.ArrayAdapter;
+import android.widget.EditText;
+import android.widget.ImageButton;
+import android.widget.Toast;
+
+import com.google.android.gms.maps.model.Marker;
```

Comment:
@cagryInside We should only import the Maps API v2 objects in the maps package.  So, we need to figure out a way to remove these imports and modify BaseMapFragment so you can add the marker from this class but by passing in non Maps API v2 objects.